src/logger/cloudwatch.logger.ts

```diff
@@ -0,0 +1,91 @@
+import AWS from "aws-sdk"
+import Bluebird from "bluebird"
+import winston from "winston"
+import WinstonCloudwatch from "winston-cloudwatch"
+
+import { consoleLogger } from "./console.logger"
+
+// AWS
+const AWS_REGION_NAME = "ap-southeast-1"
```

Comment:
nit: shall we use env var for this?

Reply:
nit so can dismiss this but 
1) we already have an env var for the region
2) we could move to a more consistent code base that way

Reply:
cc @harishv7 @alexanderleegs should this be env var or const? idm either (since our env var is now handled thru `convict`) but cos now codebase inconsistent, so will raise another PR to do this (make all either env var or ref the `const`)

Reply:
@seaerchin Either is fine, but it's should be exposed as a configurable parameter. Since we alr have it in env, it is good to standardise towards using that

Reply:
sure, gonna raise another PR for this